spec/requests/webhooks_spec.rb

```diff
@@ -152,4 +152,72 @@
       end
     end
   end
+
+  describe 'POST /adyen' do
+    let(:organization) { create(:organization) }
+
+    let(:adyen_provider) do
+      create(:adyen_provider, organization:)
+    end
+
+    let(:adyen_service) { instance_double(PaymentProviders::AdyenService) }
+
+    let(:body) do
+      path = Rails.root.join('spec/fixtures/adyen/webhook_authorisation_response.json')
+      JSON.parse(File.read(path))
+    end
+
+    let(:result) do
+      result = BaseService::Result.new
+      result.body = body
+      result
+    end
+
+    before do
+      allow(PaymentProviders::AdyenService).to receive(:new)
+        .and_return(adyen_service)
+      allow(adyen_service).to receive(:handle_incoming_webhook)
+        .with(
+          organization_id: organization.id,
+          body: body["notificationItems"].first&.dig("NotificationRequestItem")
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)